src/app/inventory/store/d2-item-factory.ts

```diff
@@ -676,10 +680,10 @@ export function makeItem(
 
   // Secondary Icon
   if (createdItem.sockets) {
-    const multiEmblem = createdItem.sockets.allSockets.filter(
+    const multiEmblem = createdItem.sockets.allSockets.find(
       (socket) => socket.plugged?.plugDef.itemType === DestinyItemType.Emblem
     );
-    const selectedEmblem = multiEmblem[0]?.plugged;
+    const selectedEmblem = multiEmblem?.plugged;
 
     if (selectedEmblem) {
       createdItem.secondaryIcon = selectedEmblem.plugDef.secondaryIcon;
```

Comment:
FWIW I think Emblems stopped having sockets / being plugs years ago, so we could save some cycles here.